.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -1,5 +1,6 @@
 <component name="ProjectCodeStyleConfiguration">
   <state>
     <option name="USE_PER_PROJECT_SETTINGS" value="true" />
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="GoogleStyle" />
```

Comment:
I'm not quite sure how this managed to sneak into the PR....

Reply:
You are truly right sorry about that